lib/fog/google/models/pubsub/subscription.rb

```diff
@@ -0,0 +1,90 @@
+require "base64"
+require "fog/core/model"
+require "fog/google/models/pubsub/received_message"
```

Comment:
I took out all three and the tests/examples passed.  I don't think they are required.

I'm curious though why we should remove them - my understanding was that require wouldn't load a file more than once (I'm still relatively new to ruby so I am likely missing something obvious)